src/ripple/rpc/impl/KeypairForSignature.h

```diff
@@ -33,6 +34,7 @@ struct KeyPair
 };
 
 KeyPair keypairForSignature (Json::Value const& params, Json::Value& error);
```

Comment:
I'd feel more comfortable with a `pair` when the members are different types, so a mismatch is caught at compile time.